pkg/cmd/indices/list/list.go

```diff
@@ -43,6 +43,9 @@ func NewListCmd(f *cmdutil.Factory) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return runListCmd(opts)
 		},
+		Annotations: map[string]string{
```

Comment:
Only added this one and `search.go` for testing, will add it to all other commands that can be run with the web CLI with the second PR that'll take care of standardize examples